### PR TITLE
fix(dialog): fix dialog position or overflow menu

### PR DESCRIPTION
**Pull request by DusanMilko**
Open to feedback and discussion.
I know you can pass an offset to the component but it seems wrong for the component to break by default because some parent has some position set with some modifying positions. It should work as expected by default.

Fixes: https://github.com/IBM/carbon-components-angular/issues/1557

I've even tested this with multiple nesting elements: https://codepen.io/dusanmilko/pen/pobzbqR

Before:
Screen Shot 2020-10-05 at 10 50 29 AM

After:
Screen Shot 2020-10-05 at 10 50 40 AM

Before:
Screen Shot 2020-10-05 at 10 51 36 AM

After:
Screen Shot 2020-10-05 at 10 51 22 AM

### CODE REVIEW

File: src/dialog/overflow-menu/overflow-menu-pane.component.ts [hunk 1]
```diff
@@ -70,6 +74,30 @@ export class OverflowMenuPane extends Dialog implements AfterViewInit {
...
+	}
+
+	matchesAttr(el, attr, val) {
+		const styles = window.getComputedStyle(el);
+
+		if (val.includes(styles[attr])) {
+			return true;
+		} else {
+			return false;
+		}
```
Comment (theiliad) [inline src/dialog/overflow-menu/overflow-menu-pane.component.ts:98]: how about `return val.includes(styles[attr])`?

**Review by theiliad — CHANGES_REQUESTED** [review]

File: src/dialog/overflow-menu/overflow-menu-pane.component.ts (hunk 1 above)
Comment (DusanMilko) [reply src/dialog/overflow-menu/overflow-menu-pane.component.ts]: Updated, great feedback!

**Review by cal-smith — APPROVED** [review]
Nice! We'll have to integrate this into the upstream position utility at some point, but for now this looks great! 🎉

**Comment by DusanMilko** [comment]
> Nice! We'll have to integrate this into the upstream position utility at some point, but for now this looks great! 🎉

Yeah, agreed. I'll also apply this to other components that have a similar issue like dropdown and etc.

**Comment by cal-smith** [comment]
Sounds good

**Pull request merged**